ch02_Context.md

```diff
@@ -103,8 +103,8 @@ De standaard NL-LOM beschrijft op welke
 manier metadata moet worden vastgelegd bij educatief materiaal. Een
 voorziening dat gebruik maakt van NL-LOM is Edurep, een zoekmachine
 die op basis van metadata educatief materiaal vindbaar maakt. 
-([NL-LOM](https://www.forumstandaardisatie.nl/standaard/nl-lom);
-[Edurep](https://www.kennisnet.nl/edurep/))
+(<a href=https://www.forumstandaardisatie.nl/standaard/nl-lom>NL-LOM</a>;
```

Comment:
We houden in de markdown files markdown syntax aan voor [hyperlinks](http://hyperlink), het is geen html (maar het gebruikt wel hier en daar html)

Reply:
Heeft u de werkversie bekeken? De markdown links worden niet goed gerenderd (waarschijnlijk vanwege het aside block) op mijn browser (edge Version 127.0.2651.98, werklaptop).

![image](https://github.com/user-attachments/assets/c680e197-bc98-4781-9076-0c17f4cc0e41)

Reply:
Er is een lege regel nodig om terug te schakelen naar Markdown. Voorbeeld van MD-stijl-link in `<aside>`:
https://github.com/Logius-standaarden/BOMOS-Fundament/blob/956b728519297967aa301e821bb9f0e3c0800410/ch03_BOMOS_gebruiken.md?plain=1#L76-L80

Reply:
zo beter?